### DISCUSSION TITLE
The new Zowe doc site framework is live!  🎉

### DISCUSSION THREAD

**Discussion by nannanli** [zowe/docs-site#1742] [Show and tell]
Today we launched our new Zowe doc site framework that will bring you an enhanced content experience. All the improvements are made based on community feedback on the site. We walked through all the issues and feedback over the past 3 years in the Zowe `docs-site` GitHub repo. After research and design discussion, we decided to migrate the site's static generator from Vuepress to Docusaurus and revamp the overall experience for you to consume Zowe content.

We want to hear your feedback on how you think of the current experience and more importantly how we can make the documentation more engaging and helpful. **Share with us any feedback and ideas here in this discussion thread. You can also reach out to the Zowe doc squad in [Slack](https://openmainframeproject.slack.com/archives/CC961JYMQ).**

Check out a list of new features and improvements as follows. (Special thanks to our contributors @idontknowjs  @PlutoZhang for making this happen!) 

-  **Three column layout (nav-topic-mini toc)** 
   ![image](https://user-images.githubusercontent.com/14340789/125296809-3d998d80-e359-11eb-9ec4-c7d807ea5b02.png)

-  **Collapsible sidebar (can be changed into two column layout as per need)**        
   ![1*RHiaKmgqkLeNQKjTNWPOBA](https://user-images.githubusercontent.com/14340789/125297046-789bc100-e359-11eb-89fb-2caffc3a410d.gif)

-  **Two themes (dark and light)**   
   ![1*Px8GtNG11xup2cZaRZdTJQ](https://user-images.githubusercontent.com/14340789/125297209-9ff28e00-e359-11eb-8a29-20cb03e5fd6a.gif)

-  **Zoom-in option for small Doc Images**   
   ![1*E5x0t68RSf6BN3X-3TPxAg](https://user-images.githubusercontent.com/14340789/125297438-d0d2c300-e359-11eb-9a65-9810d39a29dd.gif)

- Estimated reading time for each page   
   ![image](https://user-images.githubusercontent.com/14340789/125297159-949f6280-e359-11eb-848b-59f8d2311fa3.png)

- Support reading docs offline   
- Social module: "Share" button that allows you to share a topic to social platforms
-  Feedback module
   -  Rate "Is this topic helpful" - thumbs up and thumbs down. Helpful or not helpful. When not helpful, you can get a link to create a GitHub issue. 
   -  "Open doc issue" option for each page that directs users to the doc site repo with associated topic URL and background information in the issue automatically.  
   -  Better visible "Edit this topic" link that directs users to the source file for each topic
- Can print PDF (download the complete doc into PDF)
- Similar look and feel with zowe.org (includes footer) 
- Enhanced doc site UX with improved home page design  
   ![FireShot Capture 007 - Zowe Docs - docs zowe org](https://user-images.githubusercontent.com/14340789/125298226-8bfb5c00-e35a-11eb-9781-9aab653ed391.png)

**Comment by mv-nbfc** [comment]
Love the Zowe documentation site. I managed to get Zowe up and running for most functions in a days time while still attending meetings - well done.
